front/superzol-app/src/components/form-item.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import {Grid, Box, Typography} from "@mui/material";
+
+const Form = ({title, func, children}) => {
```

Comment:
You are not really using the func here so I guess you can remove it from the Form unless you think we should move the button to be inside the form.